parlai/mturk/core/mturk_manager.py

```diff
@@ -14,6 +14,15 @@
 import errno
 import requests
 
+# TODO uncomment once stable is created.
+# logging.warn(
+#     'Directly importing parlai.mturk.core.<module> is pending deprecation, '
+#     'please update your callsites to use either '
+#     'parlai.mturk.core.stable.<module>  '
+#     'or parlai.mturk.core.legacy.<module>. \n'
```

Comment:
maaaaaaaybe "legacy" is too generic?

Reply:
It's trying to refer to the legacy version of core vs the stable version of core. I'm not sure what would be better though I get what you're saying. Perhaps `original_impl`?